packages/palette/src/elements/ModalDialog/ModalDialog.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import {
+  ModalDialogBackdrop,
+  ModalDialogBackdropProps,
+} from "./ModalDialogBackdrop"
+import { ModalDialogDialog, ModalDialogDialogProps } from "./ModalDialogDialog"
+import { splitBoxProps } from "../Box"
+
+export type ModalDialogProps = ModalDialogBackdropProps & ModalDialogDialogProps
+
+export const ModalDialog: React.FC<ModalDialogProps> = ({
+  children,
+  footer,
+  hasLogo,
+  onClose,
+  title,
+  ...rest
+}) => {
+  const [boxProps, modalProps] = splitBoxProps(rest)
+
+  return (
+    <ModalDialogBackdrop onClose={onClose} {...modalProps}>
+      <ModalDialogDialog
```

Comment:
Since it's used internally it's not an issue, but the name sounds redundant ♻️

Reply:
Yeah, I dislike this but am reaching for alternatives. I wish I could just take over the `Dialog` component. We have to keep these exports namespaced, regardless.

Reply:
`ModalDialogContent`? `ModalDialogWindow`?

Reply:
`ModalDialogContent` works for me. Will update.